src/components/grid-aware/StatsBlock/StatsBlock.tsx

```diff
@@ -32,7 +35,10 @@ const StatsBlock = ({ title, statCards, theme }: StatsBlockProps) => (
     />
     <div className={s.bleedMainContent}>
       <section className={s.gridParent}>
-        <h1 className={s.title}>{title}</h1>
+        <h1 className={`${s.title} ${subtitle ? s.impactTitle : ""}`}>
```

Comment:
Related to my comment about renaming the `.impactTitle` CSS class to something generic like `.blue`, could you make the conditional expression dependent on an explicit prop on the StatsBlock, similar to our `theme` props? I think it's better to be explicit here about which color we want the title to be than to implicitly decide on it based on the presence or absence of a subtitle.